### ISSUE TITLE
steps per epoch 192000 for training

### ISSUE THREAD

**Issue by dovanhuong** [Wongi-Choi1014/Korean-OCR-Model-Design-based-on-Keras-CNN#16]
Dear @Wongi-Choi1014 ,

 I completed to use your script to re-train the algorithm. However, it seem the result my output is not same as you with the highest result can achieve 83% of accuracy. As I see, the batch size you used 128 and the log you got with 192000 steps per epoch, for my case it only have 1500 steps per epoch. 
=> My question is how can you have 192000 steps per epoch? As I see the dataset our case have overall 264385 images.

Thanks you so much,
